platform/ios/src/UIViewController+MGLAdditions.h

```diff
@@ -0,0 +1,11 @@
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface UIViewController (MGLAdditions)
+
++ (UIViewController *)mgl_topMostViewControllerInHierarchyWith:(UIViewController *)viewController;
```

Comment:
In Objective-C, a selector piece (`mgl_topMostViewControllerInHierarchyWith:`) [should always end with a noun](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html#//apple_ref/doc/uid/20001282-1001751), not a preposition.

This should be an instance method or read-only property named `mgl_topmostViewController`. The return value is related to `viewController`, not the UIViewController class as a whole. One might expect this class method, as written, to be a factory method for creating UIViewControllers.